### ISSUE TITLE
confusing interaction with icons for each experience

### ISSUE THREAD

**Issue by jpbonjour** [zdmc23/paraguide-public#6]
1. Launch the app
2. Explore the Drive, Walk, Transit, and Mobile icons at the top of each "experience" 

Outcome:
Completely different experiences happen for each: Waze, default map view that doesn't do anything, apple maps but does not default to transit mode, and third party ride sharing categories that then go to their own app or web site. 
![img_5585](https://user-images.githubusercontent.com/41528698/47966793-647d5e80-e00b-11e8-8040-4d81f9094815.PNG)

**Comment by zdmc23** [comment]
Yeah, we decided on launching Waze by default (if it's installed) with the assumption that if you installed Waze, then you probably would prefer to use it for Maps.  The Walking Map would work for you if you were in NYC.  It offers in-app turn-by-turn navigation that you just have to swipe for, with audio.  The Ridesharing services offer (at least for Uber) deep links that will drop you in the context of that app with directions and allow you to order your ride immediately.  We'll be doing the same with the others as well.

How do you choose?  It's tempting to go with Apple Maps for all, but we want to keep users in-app as much as possible (so we have the turn-by-turn walking, if you're in that location).  Transit almost has to be Apple Maps.  So do we throw out Waze?  I don't mind, bc I personally don't use Waze, but not sure how strongly Waze users feel about it vs. Apple Maps
